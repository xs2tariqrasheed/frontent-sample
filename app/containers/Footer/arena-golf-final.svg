<svg xmlns="http://www.w3.org/2000/svg" width="107" height="65" viewBox="-1 0 110 62">
  <defs>
    <linearGradient id="vr2jga1qma" x1="50%" x2="50%" y1="58.611%" y2="43.636%">
      <stop offset="0%" stop-color="#26225D"/>
      <stop offset="100%" stop-color="#3A348F"/>
    </linearGradient>
    <linearGradient id="jdm95zwp7b" x1="50.126%" x2="50.126%" y1="79.86%" y2="-54.493%">
      <stop offset="0%" stop-color="#FFF"/>
      <stop offset="100%" stop-color="#FFC430"/>
    </linearGradient>
  </defs>
  <g fill="none">
    <path fill="url(#vr2jga1qma)" stroke="#E7E6FF" stroke-linejoin="round" stroke-width="1.638" d="M53.49-.819c2.647 0 5.294.781 7.58 2.344 2.284 1.562 3.973 3.746 4.933 6.213.855 2.196 1.134 4.616.744 7.023 5.429.693 10.785 1.871 16.003 3.522h0v-2.4c.352.086.703.176 1.048.276.611.178 1.22.392 1.821.605l-.582-.204v-4.522h1.63l9.857 1.765-9.112 1.811v1.796c.774.284 1.538.576 2.277.888h0v2.835h8.928l9.276 29.81-8.579 7.83H76.156l-2.118 1.928H32.569l-2.118-1.927H7.552l-8.486-7.832 9.112-29.81h8.931v-2.815c.748-.355 1.514-.634 2.289-.915l-.007.002v-5.366h1.629l9.856 1.763-9.083 1.813v.953c.749-.254 1.508-.5 2.287-.725h0v2.462c5.266-1.667 10.672-2.853 16.152-3.545-.39-2.406-.112-4.826.743-7.021.96-2.467 2.649-4.65 4.934-6.213C48.194-.038 50.842-.82 53.49-.82z"/>
    <circle cx="53.478" cy="12.655" r="9.297" fill="#FFF"/>
    <path fill="#B0AEC2" d="M44.28 11.208c.11.662.293 1.31.545 1.932-.028.05-.035.102-.037.155v.053c-.037.437.193.85.576 1.054l.078.038h.038c.066.022.134.038.202.049.448.171.73.614.7 1.092l.001.076-.004.142.002.065c-.025.282.062.561.244.778.18.216.441.35.723.374.073.012.148.015.222.012l.11-.012.098-.024c.099-.02.2-.027.301-.02.48.05.865.407.958.873l.014.088.004.084c.036.389.283.73.646.883.361.152.775.093 1.078-.149l.068-.059.075-.062c.206-.154.463-.228.722-.205.346.033.654.229.832.524l.041.076.036.078c.169.327.499.545.87.566.442.046.861-.188 1.056-.58l.036-.08.042-.079c.15-.254.4-.435.692-.497.32-.069.654.015.904.226.175.169.403.27.644.29.281.026.561-.062.778-.243.192-.162.32-.385.362-.63l.012-.093v-.05l.012-.083c.096-.548.593-.941 1.157-.9.057.006.113.015.169.03l.082.025.092.021.093.012c.582.036 1.09-.392 1.152-.972.004-.078 0-.156-.01-.234l-.023-.115-.01-.1c-.003-.045-.002-.09.002-.134l.008-.066c.036-.415.312-.77.705-.906.04-.008.079-.017.117-.03l.057-.02.08-.033c.342-.159.575-.489.608-.867.013-.145-.006-.29-.054-.426.237-.612.41-1.247.513-1.895.054.47.074.942.06 1.414 0 5.135-4.163 9.297-9.297 9.297-5.135 0-9.297-4.162-9.297-9.297 0-.484.037-.968.114-1.446zm8.476 5.44c.317-.32.801-.405 1.208-.213.408.192.65.62.606 1.067-.01.124-.043.244-.098.355-.16-.345-.495-.576-.874-.606-.442-.04-.862.202-1.048.606-.194-.406-.112-.89.206-1.21zm-2.34-.68l.005.027c.42.026.787.29.945.678.158.39.08.835-.203 1.146l.002-.036-.002-.035c0-.603-.488-1.092-1.091-1.092-.28.002-.547.112-.748.306.048-.536.479-.953 1.007-.991l.085-.003zm6.048 0c.56-.001 1.025.434 1.06.994-.305-.304-.762-.393-1.159-.227-.396.166-.652.556-.649.986v.07c-.2-.202-.314-.474-.316-.758.014-.582.482-1.05 1.064-1.065zm-8.442-1.618l.09.008c.57.102.96.632.89 1.206-.041.297-.208.562-.459.726.033-.08.056-.165.071-.25.06-.415-.13-.826-.484-1.049-.354-.223-.806-.217-1.154.016.18-.416.598-.675 1.046-.657zm11.99.657c-.347-.233-.8-.24-1.153-.016-.354.223-.544.634-.484 1.048.012.087.036.171.071.251-.367-.243-.546-.687-.45-1.117.098-.43.45-.754.887-.815h-.006c.481-.06.943.205 1.136.65zm-7.285-1.483c.37-.253.859-.253 1.23 0 .37.253.549.707.45 1.145.008.072.008.146 0 .218-.232-.345-.622-.55-1.038-.546-.425-.013-.828.193-1.065.546-.017-.071-.026-.145-.027-.218-.1-.438.079-.892.45-1.145zm-2.538-.485c.392-.027.767.165.975.497.208.333.217.754.021 1.095-.032.066-.072.128-.12.185-.057-.41-.315-.764-.688-.944-.365-.194-.803-.194-1.168 0 .02-.072.048-.141.082-.208.161-.358.506-.598.898-.625zm6.365 0c.392.027.737.267.898.625.032.068.06.137.082.208-.365-.194-.803-.194-1.168 0-.373.18-.63.534-.688.944-.047-.057-.088-.119-.12-.185-.195-.341-.187-.762.021-1.095.209-.332.583-.524.975-.497zm4.349-.838c.336 0 .653.16.851.432-.164-.099-.354-.148-.546-.142-.577.026-1.034.497-1.042 1.075.003.228.077.45.213.633-.42-.241-.624-.737-.496-1.204.128-.468.557-.79 1.042-.783l-.022-.01zm-14.555 0c.485-.007.914.315 1.042.783.128.467-.075.963-.496 1.204.136-.183.21-.405.213-.633-.014-.592-.499-1.065-1.092-1.064-.191-.006-.38.043-.545.142.18-.247.458-.402.76-.428l.118-.004zm7.043-1.354c.588 0 1.064.477 1.064 1.065.009.072.009.146 0 .218-.214-.318-.562-.518-.941-.543l-.096-.003c-.434-.02-.848.186-1.092.546l-.004-.109.004-.11c0-.587.477-1.064 1.065-1.064zm-7.48-1.255l.006.027c.485-.007.913.315 1.042.783.128.467-.076.962-.496 1.204.136-.183.21-.405.208-.633 0-.593-.473-1.077-1.065-1.092-.191-.001-.38.048-.546.142.181-.246.459-.4.76-.427l.092-.004zm14.929.002l.07-.002h-.007c.336 0 .653.16.851.431-.436-.253-.99-.164-1.325.214-.334.378-.355.94-.05 1.342-.42-.242-.624-.737-.496-1.205.12-.44.507-.75.957-.78z"/>
    <path fill="#FFF" d="M58.348 50.427v.098c-.104 0-.17.027-.191.055-.027.023-.04.057-.039.093v3.062l2.834-.022c.038.002.074-.014.098-.043.03-.058.046-.121.05-.186h.097v1.414H61.1c0-.057-.017-.112-.049-.158-.032-.01-.066-.01-.098 0H56.66v-.099c.104 0 .17-.027.19-.054.027-.024.041-.058.04-.093v-3.821c.001-.036-.013-.07-.04-.093l-.042-.018c-.035-.014-.089-.03-.148-.037v-.098h1.687zm-18.998-.05c.102.006.204.017.306.033l.3.06.371.099c.086.023.17.05.254.084l.123.052c.069.036.142.061.218.077.047-.002.09-.026.115-.066l.098.033-.546 1.092-.098-.06c.004-.03.004-.059 0-.088 0-.038-.05-.082-.142-.125-.111-.057-.229-.1-.35-.131-.158-.041-.318-.074-.48-.098-.18-.028-.363-.04-.546-.039-.184.002-.368.027-.546.077-.172.049-.333.13-.475.24-.14.11-.255.25-.333.41-.081.17-.122.357-.12.545-.004.201.047.4.148.574.094.156.22.29.37.393.153.105.323.183.503.229.178.048.361.074.546.076.206 0 .413-.014.617-.043.184-.026.367-.065.546-.115v-.546l-1.01.006c-.037-.004-.072.016-.088.049-.018.035-.03.074-.032.114H39v-1.146h.099c0 .098.049.147.12.147h2.418v.099c-.037-.002-.073.012-.099.04-.017.018-.027.04-.031.065v1.418c-.007.032-.007.065 0 .098 0 .022.024.033.056.037l.063.001v.098c-.817.41-1.722.613-2.637.59-.335.003-.669-.04-.993-.126-.305-.076-.593-.212-.846-.398-.248-.179-.452-.412-.595-.682-.138-.262-.214-.551-.224-.846v-.126c-.01-.336.067-.67.224-.967.143-.271.347-.506.595-.687.253-.19.54-.33.846-.41.323-.092.657-.138.993-.136h.36zm9.613-.021c.335-.001.668.047.989.142.293.089.567.231.807.42.23.187.417.421.546.688.27.605.27 1.295 0 1.9-.129.266-.315.501-.546.688-.238.19-.513.329-.807.41-.648.174-1.33.174-1.977 0-.293-.083-.568-.222-.808-.41-.23-.186-.417-.421-.546-.688-.276-.603-.276-1.297 0-1.9.13-.267.316-.502.546-.688.242-.188.516-.33.808-.42.321-.095.654-.143.988-.142zm21.733-.175v1.425l-.12-.005c-.003-.056-.02-.11-.049-.159-.018-.02-.043-.034-.07-.038h-2.73v.726h1.818c.037.004.073-.01.098-.038.03-.044.047-.095.049-.147h.098v1.375h-.098c-.002-.063-.019-.125-.05-.18-.023-.03-.06-.045-.097-.044h-1.818v1.382c-.002.034.01.067.032.092.06.037.128.056.197.055v.098h-1.72v-.098c.068.001.134-.018.191-.055.033-.02.055-.054.06-.092v-3.822c0-.035-.013-.068-.038-.093-.057-.036-.123-.055-.19-.054v-.099h4.192c.038.002.074-.014.098-.043.03-.058.046-.121.05-.186h.097zm-21.733 1.174c-.174 0-.347.027-.513.082-.155.052-.299.131-.426.235-.123.103-.224.231-.294.376-.075.159-.112.333-.11.508-.002.177.035.352.11.513.07.145.17.274.294.377.127.103.271.183.426.235.334.11.693.11 1.027 0 .155-.052.299-.132.425-.235.124-.103.225-.232.295-.377.075-.16.112-.336.11-.513.002-.175-.035-.349-.11-.508-.07-.145-.17-.273-.295-.376-.126-.104-.27-.183-.425-.235-.166-.055-.34-.083-.514-.082zm-18.157.393l.06.033v1.496l-.054.032-4.095-.748v-.081l4.09-.732zm45.344-.033l4.09.748v.082l-4.095.748-.055-.033v-1.512l.06-.033zM18.354 25.79c.857.457 1.423 1.247.94 3.346l-.81 3.297c-.42 1.698-1.195 2.9-2.8 3.51 1.244.599 1.472 1.746 1.101 3.363l-.997 4.046c-.546 2.222-1.638 3.04-2.915 3.51h-8.21l6.594-21.072h7.097zm77.133 0l1.392 4.367h-3.986l1.092 3.821h3.854l.532 1.698.85 2.67h-3.975l1.18 4.1h4.454l1.392 4.367H93.49L87.997 25.79h7.49zm-69.976 0L22.394 40.9h3.772l-.72 3.975h-7.644l4.433-19.085h3.276zm11.633 0c.857.425 1.48 1.168 1.261 3.182-.45 4.244-.904 8.484-1.359 12.72-.218 2.015-1.092 2.73-2.184 3.183h-5.497l-.154-.072c-.882-.438-1.466-1.205-1.145-3.11l2.129-12.72c.338-2.015 1.168-2.73 2.14-3.183h4.81zm5.34 0h4.809l.155.078c.812.44 1.408 1.208 1.352 3.104l-.148 4.613h-3.455l.229-4.083-.197-.268h-1.053l-.23.268-.78 11.66.213.268h1.146l.246-.267.224-4.03h3.57l-.03.645-.03.703c-.029.78-.045 1.443-.073 2.27l-.036.941c-.066 2.015-.885 2.73-1.938 3.183H40.96l-.168-.076c-.91-.439-1.546-1.208-1.371-3.107.386-4.24.775-8.48 1.168-12.72.18-2.014.955-2.73 1.894-3.182zm11.813-.022l.004 1.556.01 1.436c.014 1.373.035 2.63.052 4.432h.43l.344-1.49.39-1.735c.286-1.285.545-2.498.905-4.2h3.002l.131 2.867-.343 1.451c-.425 1.78-.744 3.053-1.295 5.17l.74 2.252c.478 1.438.895 2.647 1.542 4.485l.131 2.861h-3.428l-.606-2.147-.383-1.387c-.344-1.26-.665-2.479-1.096-4.153l-.415.022.07 7.687h-3.76l.3-19.107h3.275zm10.143.022l1.278 15.11h3.777l.437 3.975H62.29L61.165 25.79h3.275zm12.01 0l.694 3.974H73.54l.471 3.164.042.286h3.472l.688 3.974h-3.57l.545 3.713h4.002l.699 3.974h-7.867L69.665 25.79h6.785zm9.832 0l.989 3.974h-2.435c1.157 5.037 2.316 10.074 3.477 15.11h-3.761c-1.034-5.036-2.07-10.073-3.106-15.11H79.01l-.748-3.974h8.02zm-73.42 12.294h-1.485L9.87 43.352h1.54l.338-.295 1.283-4.684-.122-.212-.047-.077zm21.962-8.85h-1.092l-.123.136-.08.084-.048.048-1.638 11.66.191.268h1.152l.262-.267 1.524-11.661-.148-.268zm-19.538.066h-1.398l-1.507 5.273h1.453l.322-.294 1.283-4.684-.153-.295z"/>
    <path fill="url(#jdm95zwp7b)" d="M16.885 38.684c-.024.174-.053.347-.09.519l-.057.256-.961 3.893c-.535 2.156-1.574 2.991-2.813 3.478l-.113.043H4.662l1.73-5.53c1.148-.361 2.32-.704 3.516-1.027l.901-.24-.939 3.276h1.54l.338-.295.29-1.036c.24-.872.424-1.571.66-2.42 1.376-.327 2.772-.633 4.187-.917zm74.54.339c1.456.313 2.884.651 4.285 1.015l.7 2.43h4.47l1.414 4.394H93.49c-.677-2.613-1.365-5.226-2.064-7.84zm-68.338-1.398l-.677 3.276h3.772l-.25 1.374c-.075.414-.143.801-.216 1.22l-.243 1.407H17.83l1.534-6.66c.93-.168 1.862-.324 2.793-.471l.93-.146zm27.782-2.293h2.956l.862.005c1.026.007 2.121.025 3.268.055l.751 2.288c.456 1.372.866 2.562 1.492 4.334l.131 2.86h-3.428l-.606-2.146c-.452-1.618-.803-2.925-1.28-4.77l-.2-.77h-.403l.056 5.217c.004.523.007 1.074.008 1.685l.001.763h-3.756l.148-9.52zm-13.2.623c-.208 1.914-.414 3.826-.618 5.737-.217 1.954-1.04 2.71-2.085 3.143l-.098.04h-5.503l-.157-.074c-.85-.425-1.427-1.16-1.163-2.914l.845-5.075c1.216-.146 2.422-.279 3.62-.399l-.683 4.755.191.268h1.157l.263-.268.65-4.913c1.211-.11 2.405-.21 3.58-.3zm5.84-.36l-.37 5.568.104.133.108.134h1.147l.245-.267.224-4.03h3.57l-.141 4.56c-.064 1.954-.837 2.711-1.846 3.143l-.092.039H40.96l-.168-.076c-.883-.426-1.508-1.161-1.39-2.937l.013-.17.546-5.896c1.22-.08 2.402-.147 3.549-.202zm18.245-.077c.917.04 1.857.087 2.818.14l.725.04.442 5.203h3.778l.437 3.974h-7.643c-.19-3.12-.375-6.238-.557-9.357zm9.215.578c2.348.197 4.766.443 7.217.743l.039.17c.006.03.011.058.015.088l.006.091h-3.57l.34 2.338.206 1.375h3.974l.7 3.974h-7.873c-.36-2.926-.712-5.852-1.054-8.779zm12.07 1.398c.828.124 1.655.255 2.48.393l1.238.213 1.567 6.775-3.772-.011c-.506-2.457-1.01-4.913-1.512-7.37z"/>
    <path fill="#262261" stroke="#D6D3C1" stroke-linejoin="round" stroke-width=".546" d="M101.765 55.937l.88 1.231.392-1.003.439 1.723-.442.1-.255.623-.136-.268v2.965l-.878-5.37zm.115 1.187l.147.78-.536.102-.147-.781.536-.101z"/>
  </g>
</svg>
