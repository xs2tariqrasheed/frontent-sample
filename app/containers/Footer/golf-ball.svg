<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="57" height="58" viewBox="0 0 57 58">
    <defs>
        <path id="prefix__a" d="M0 0.094L56.621 0.094 56.621 57.709 0 57.709z"/>
        <path id="prefix__c" d="M0 0.225L56.621 0.225 56.621 33.518 0 33.518z"/>
    </defs>
    <g fill="none" fill-rule="evenodd">
        <g transform="translate(0 .197)">
            <mask id="prefix__b" fill="#fff">
                <use xlink:href="#prefix__a"/>
            </mask>
            <path fill="#FEFEFE" d="M56.621 28.902c0 15.91-12.675 28.807-28.31 28.807C12.675 57.709 0 44.81 0 28.902 0 12.992 12.675.094 28.31.094c15.636 0 28.311 12.898 28.311 28.808" mask="url(#prefix__b)"/>
        </g>
        <g transform="translate(0 24.388)">
            <mask id="prefix__d" fill="#fff">
                <use xlink:href="#prefix__c"/>
            </mask>
            <path fill="#D5D2C0" d="M56.278.225c-.313 2.037-.844 3.998-1.555 5.868.135.412.192.858.153 1.318-.11 1.31-.959 2.37-2.092 2.804l-.004.007c-.014.002-.027.007-.041.01-.174.063-.354.111-.538.143-1.16.419-2.037 1.49-2.148 2.82-.027.317-.006.627.052.924l-.01.012c.087.346.122.712.09 1.088-.152 1.814-1.72 3.159-3.502 3.004-.192-.017-.377-.053-.557-.102l-.023.02c-.243-.084-.499-.143-.766-.167-1.782-.155-3.35 1.19-3.503 3.004-.002.025 0 .05-.002.075l-.016.008c-.002.048-.001.095-.006.143-.151 1.813-1.72 3.158-3.502 3.003-.767-.066-1.447-.4-1.962-.903-.5-.44-1.13-.735-1.838-.797-1.385-.12-2.638.667-3.2 1.875l-.014.001c-.516 1.31-1.827 2.183-3.281 2.056-1.262-.11-2.29-.942-2.73-2.061l-.02-.002c-.479-1.02-1.46-1.766-2.647-1.869-.925-.08-1.79.246-2.432.83l-.017-.007c-.646.614-1.533.96-2.483.877-1.66-.144-2.919-1.538-2.961-3.193l-.02-.01c-.14-1.56-1.359-2.843-2.947-2.98-.424-.038-.836.013-1.22.132l-.011-.01c-.325.078-.666.107-1.016.077-1.782-.155-3.104-1.75-2.952-3.564.012-.142.036-.28.064-.414l-.006-.007c.037-.156.066-.316.08-.48.127-1.518-.78-2.877-2.122-3.372-.215-.033-.422-.087-.62-.16-.041-.007-.08-.016-.122-.02l-.017-.032C2.548 9.638 1.708 8.316 1.831 6.85c.018-.218.058-.427.115-.63C1.21 4.314.663 2.309.343.226.12 1.687 0 3.185 0 4.711c0 15.91 12.675 28.807 28.31 28.807 15.636 0 28.31-12.898 28.31-28.807 0-1.526-.118-3.024-.342-4.486" mask="url(#prefix__d)"/>
        </g>
        <path fill="#D5D2C0" d="M28.586 43.236c1.196.104 2.182.858 2.657 1.89.157-.341.261-.715.294-1.112.153-1.814-1.169-3.41-2.951-3.564-1.783-.155-3.35 1.19-3.503 3.003-.05.597.062 1.169.295 1.674.559-1.216 1.816-2.011 3.208-1.89M18.878 39.27c-1.718 0-3.119 1.362-3.228 3.082.584-.588 1.388-.952 2.275-.952 1.788 0 3.239 1.476 3.239 3.296 0 .072-.006.143-.01.215.594-.598.963-1.427.963-2.344 0-1.821-1.45-3.296-3.239-3.296M37.312 39.27c1.717 0 3.119 1.362 3.228 3.082-.585-.588-1.388-.952-2.275-.952-1.789 0-3.239 1.476-3.239 3.296 0 .072.006.143.01.215-.594-.598-.963-1.427-.963-2.344 0-1.821 1.45-3.296 3.239-3.296M11.868 34.28c-1.5-.222-2.905.642-3.456 2.006.643-.441 1.442-.653 2.27-.53 1.77.261 2.998 1.933 2.741 3.735-.04.275-.114.535-.214.781.732-.5 1.263-1.3 1.4-2.258.256-1.801-.97-3.474-2.741-3.735M44.728 34.28c1.5-.222 2.904.642 3.456 2.006-.644-.441-1.442-.653-2.27-.53-1.771.261-2.998 1.933-2.741 3.735.039.275.114.535.213.781-.731-.5-1.263-1.3-1.399-2.258-.257-1.801.97-3.474 2.74-3.735M6.43 27.59c-1.064 0-2.005.524-2.596 1.331.473-.274 1.018-.433 1.6-.433 1.79 0 3.24 1.476 3.24 3.296 0 .737-.24 1.415-.642 1.964.977-.567 1.638-1.635 1.638-2.863 0-1.82-1.45-3.296-3.24-3.296M5.173 19.508c-1.065 0-2.007.525-2.597 1.331.472-.274 1.018-.432 1.6-.432 1.79 0 3.24 1.475 3.24 3.295 0 .737-.241 1.415-.643 1.964.978-.567 1.639-1.635 1.639-2.862 0-1.82-1.45-3.296-3.24-3.296M50.826 27.59c1.065 0 2.007.524 2.597 1.331-.472-.274-1.018-.433-1.6-.433-1.79 0-3.24 1.476-3.24 3.296 0 .737.24 1.415.642 1.964-.977-.567-1.638-1.635-1.638-2.863 0-1.82 1.45-3.296 3.24-3.296M27.88 34.22c1.354 0 2.534.666 3.166 1.652.047-.22.073-.448.073-.683 0-1.82-1.45-3.296-3.24-3.296-1.788 0-3.239 1.476-3.239 3.296 0 .235.026.462.073.683.632-.986 1.812-1.652 3.166-1.652M18.773 32.766c1.207.627 1.96 1.765 2.083 2.935.14-.174.265-.365.37-.574.812-1.621.18-3.606-1.414-4.433-1.593-.828-3.544-.184-4.357 1.437-.105.21-.184.424-.24.641 1.003-.585 2.351-.632 3.558-.006M50.826 19.508c1.065 0 2.007.525 2.597 1.331-.472-.274-1.018-.432-1.6-.432-1.79 0-3.24 1.475-3.24 3.295 0 .737.24 1.415.642 1.964-.977-.567-1.638-1.635-1.638-2.862 0-1.82 1.45-3.296 3.24-3.296M37.002 32.766c-1.207.627-1.96 1.765-2.084 2.935-.14-.174-.264-.365-.369-.574-.813-1.621-.18-3.606 1.414-4.433 1.593-.828 3.544-.184 4.357 1.437.104.21.183.424.24.641-1.003-.585-2.351-.632-3.558-.006M27.978 25.676c1.355 0 2.535.665 3.167 1.651.046-.22.072-.448.072-.682 0-1.82-1.45-3.296-3.239-3.296s-3.239 1.476-3.239 3.296c0 .234.026.462.072.682.633-.986 1.813-1.65 3.167-1.65"/>
    </g>
</svg>
