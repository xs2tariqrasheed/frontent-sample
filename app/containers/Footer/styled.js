import styled from 'styled-components';
import mq from '../../styles/mq';

export const FooterStyles = styled.div`
  .not-home {
    background-color: #322c8f;
  }
  .home {
    max-width: 1250px;
    background-color: #322c8f;
  }
`;

export const FooterWrapper = styled.footer`
  position: relative;

  img {
    max-width: 50px;
  }
`;

export const Container = styled.header`
  padding: 20px 20px;
  width: 100%;
  bottom: 0;
  z-index: 2;

  max-width: 1200px;
  margin: 0 auto;
`;

export const FlexContainer = styled.header`
  display: flex;
  padding: 5px 120px;
  justify-content: space-between;
  width: 100%;
  z-index: 2;

  ${mq.sm} {
    padding: 5px 10px;
  }
`;

export const Version = styled.div`
  font-family: Roboto;
  font-size: 12px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 2;
  letter-spacing: normal;
  text-align: right;
  color: #333333;
`;
