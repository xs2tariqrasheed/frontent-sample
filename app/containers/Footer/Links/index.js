/*
 * Social Media
 *
 * Social media bar
 *
 */

import React from 'react';

import { Link, Wrapper } from './styled';

export default () => (
  <Wrapper>
    <div align="right">
      <Link href="/terms-of-service">Terms of Service&nbsp;&nbsp;</Link>
      <Link href="/privacy">|&nbsp;&nbsp;Privacy Policy</Link>
    </div>
  </Wrapper>
);
