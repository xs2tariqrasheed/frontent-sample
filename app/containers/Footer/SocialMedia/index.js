/*
 * Social Media
 *
 * Social media bar
 *
 */

import React from 'react';

import { SocialLink } from './styled';

export default () => (
  <div className="inline-block align-middle">
    <div className="md:text-right">
      <SocialLink href="https://twitter.com/BlockleteGames" target="_blank">
        <i className="fab fa-twitter" />
      </SocialLink>
      <SocialLink href="https://discord.gg/hy7cRwhCfW" target="_blank">
        <i className="fab fa-discord" />
      </SocialLink>
    </div>
  </div>
);
