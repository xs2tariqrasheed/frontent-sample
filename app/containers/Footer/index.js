/*
 * Footer
 *
 * Footer for the user's account.
 *
 */

import React from 'react';
import { Link, useLocation } from 'react-router-dom';
import SocialMedia from './SocialMedia';
import { Version, FooterStyles, FooterWrapper } from './styled';
const ArenaGolfLogo =
  'https://www.blockletegames.com/images/Blocklete-B-Logo.png';

/*eslint-disable */
const Footer = () =>  {

  const location = useLocation();
  const { pathname } = location;
  const splitLocation = pathname.split('/');

  return (
    <FooterStyles>
      <div className={splitLocation[1] === '' ? 'home' : 'not-home'}>
        <FooterWrapper className="container mx-auto px-2 text-white pt-12 xs:pb-12 md:pb-20">
          <div className="text-center">
            <div className="inline-block align-middle">
              <Link to="/">
                <img
                  src={ArenaGolfLogo} alt='Blocklete Games Logo' />
              </Link>
            </div>
            <nav role="navigation" className="inline-block mx-12">
              <ul className="leading-10 inline">
                <li className="xs:mb-1 md:mb-5 md:col-span-2 inline-block mx-6 font-display"><Link
                  className="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
                  to="/">Home</Link></li>
                <li className="xs:mb-1 md:mb-5 md:col-span-2 inline-block mx-6 font-display"><Link
                  className="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
                  to={{ pathname: 'https://wbgamessupport.wbgames.com/hc/en-us' }} target="_blank">How to Play</Link></li>
                <li className="xs:mb-1 md:mb-5 md:col-span-2 inline-block mx-6 font-display"><Link
                  className="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
                  to="/terms-of-service">Terms of Service</Link></li>
                <li className="xs:mb-1 md:mb-5 md:col-span-2 inline-block mx-6 font-display"><Link
                  className="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
                  to="/playtoearn">Marketplace</Link></li>
                <li className="xs:mb-1 md:mb-5 md:col-span-2 inline-block mx-6 font-display"><Link
                  className="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
                  to="/privacy">Privacy Policy</Link></li>
              </ul>
            </nav>
            <SocialMedia />
          </div>
          <div className="text-center">
            <p className="text-sm font-display text-white tracking-wide mt-4">© 2021 Blocklete Games™. All rights reserved.</p>
          </div>
        </FooterWrapper>
      </div>
    </FooterStyles>
  );
};

export default Footer;
/* eslint-enable */
