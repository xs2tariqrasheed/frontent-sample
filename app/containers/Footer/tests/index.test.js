import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory, BrowserRouter } from 'react-router-dom';
import configureStore from '../../../configureStore';
import Footer from '../index';

describe('<Footer />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={store.store}>
        <BrowserRouter>
          <IntlProvider locale="en">
            <Footer />
          </IntlProvider>
        </BrowserRouter>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
