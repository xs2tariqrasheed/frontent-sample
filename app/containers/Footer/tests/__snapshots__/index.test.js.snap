// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render and match the snapshot 1`] = `
<footer
  class="styled__Footer-sc-1x6885q-0 cUxGfC"
>
  <footer
    class="container mx-auto px-5 text-white py-12 styled__FooterWrapper-sc-1x6885q-1 kisQZq"
  >
    <div
      class="md:grid grid-cols-12 gap-6 mb-10"
    >
      <div
        class="col-span-1 flex items-center mb-10"
      >
        <a
          href="/"
        >
          <img
            alt="Blocklete Games Logo"
            src="IMAGE_MOCK"
          />
        </a>
      </div>
      <nav
        class="col-span-12 md:col-span-6"
        role="navigation"
      >
        <ul
          class="md:grid grid-cols-2 grid-flow-row leading-10"
        >
          <li
            class="mb-5"
          >
            <a
              class="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="mb-5"
          >
            <a
              class="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
              href="/events"
            >
              Play Games
            </a>
          </li>
          <li
            class="mb-5"
          >
            <a
              class="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
              href="/faq"
            >
              How to Play
            </a>
          </li>
          <li
            class="mb-5"
          >
            <a
              class="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
              href="/playtoearn"
            >
              Marketplace
            </a>
          </li>
          <li
            class="mb-5"
          >
            <a
              class="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
              href="/terms-of-service"
            >
              Terms of Service
            </a>
          </li>
          <li
            class="mb-5"
          >
            <a
              class="hover:underline text-white no-underline text-base font-heavy uppercase tracking-wide"
              href="/privacy"
            >
              Privacy Policy (New)
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="col-span-12 md:col-start-11 md:col-span-2"
      >
        <div
          class="md:text-right"
        >
          <a
            class="styled__SocialLink-sc-1984pln-0 jcYxES"
            href="https://twitter.com/BlockleteGames"
            target="_blank"
          >
            <i
              class="fab fa-twitter"
            />
          </a>
          <a
            class="styled__SocialLink-sc-1984pln-0 jcYxES"
            href="https://discord.gg/XDJAJdr"
          >
            <i
              class="fab fa-discord"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="md:grid grid-cols-10 gap-6"
    >
      <div
        class="col-start-2 col-span-12 md:col-span-6"
      >
        <p
          class="text-sm md:text-base text-white tracking-wide"
        >
          © 2020 Blocklete Games™. All rights reserved.
        </p>
      </div>
    </div>
    <div
      class="styled__Version-sc-1x6885q-4 cObAwi"
    >
      4.0.2
    </div>
  </footer>
</footer>
`;
